draft-pauly-add-resolver-discovery.md

```diff
@@ -240,10 +240,10 @@ the client SHOULD suppress queries for Companion DoH Servers against this resolv
 or invalid response and continue to use the original Direct Resolver.
 
 The following example shows a record containing a Companion DoH URI, as returned by a query for
-the HTTPS variant of the SVCB record type on the "resolver.arpa" domain.
+the HTTPS variant of the SVCB record type for "dns://resolver.arpa":
```

Comment:
Do we still use an HTTPS record here, or is it a raw SVCB record, since the scheme is dns://?

Reply:
Good catch, yes it would be a raw SVCB record. I've issued a second commit to fix that.